workspace_tools/targets.py

```diff
@@ -165,11 +165,13 @@ def __init__(self):
         self.supported_toolchains = ["ARM", "uARM", "GCC_ARM"]
         self.default_toolchain = "uARM"
 
-class APPNEARME_MICRONFCBOARD(LPC11U34_421):
+class MICRONFCBOARD(LPC11U34_421):
     def __init__(self):
         LPC11U34_421.__init__(self)
-        self.macros = ['LPC11U34_421']
-        self.is_disk_virtual = True
+        self.macros = ['LPC11U34_421', 'APPNEARME_MICRONFCBOARD']
```

Comment:
Shouldn't this only expand LPC11U34_421? Like this for macro and extra labels, others are proably the same:

```
self.macros += ['APPNEARME_MICRONFCBOARD']
```